### PR TITLE
expose modules list

### PR DESCRIPTION
**Pull request by zimbatm**
Make the modules list available outside of flakes and prefix them.

### CODE REVIEW

**Comment by zimbatm** [comment]
I think we'll need some sort of structure like that. What do you think?

File: default.nix [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  # Profiles
+  profiles-common = import ./profiles/common.nix;
+  profiles-server = import ./profiles/server.nix;
+  profiles-desktop = import ./profiles/desktop.nix;
+
+  profiles-efi = import ./profiles/efi.nix;
+  profiles-nginx = import ./profiles/nginx.nix;
+  profiles-telegraf = import ./profiles/telegraf.nix;
```
Comment (Mic92) [inline default.nix:9]: Mhm. I find the `profiles-` prefix a bit verbose.

File: default.nix [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  # Profiles
+  profiles-common = import ./profiles/common.nix;
+  profiles-server = import ./profiles/server.nix;
+  profiles-desktop = import ./profiles/desktop.nix;
```
Comment (Mic92) [inline default.nix:5]: I think these are actually our main modules while the below behave more like mixins that you can add on top.

Comment (Mic92) [reply default.nix:5]: Something that indicates that you should add one of these and than some of the below would be nice.

Comment (Mic92) [reply default.nix:5]: On the other hand already the categories profiles and roles on its own are confusing enough..

Comment (zimbatm) [reply default.nix]: Good point. I moved things around a bit.

**Pull request merged**